test/unit/polymer-dom-elements.html

```diff
@@ -357,3 +357,245 @@
   });
   </script>
 </dom-module>
+
+
+<dom-module id="x-shadow-host-root">
```

Comment:
Make sure to always register dependent elements before an element that uses them (e.g. `x-shadow-host-root-0`)